app/components/about-header.tsx

```
@@ -1,56 +1,79 @@
-                We aspire to be a DeFi ecosystem where users can seamlessly
-                access their Ethereum assets liquidity on Starknet while keeping
-                their assets securely locked on L1.
-              </p>
-            </div>
+      <div className="flex flex-col-reverse md:flex-row p-4 justify-between items-center gap-10 md:gap-20">
+        <div className="flex flex-col w-full max-w-[721px] gap-6 sm:gap-8 md:gap-10">
+          <div className="flex flex-col gap-6 sm:gap-8 md:gap-[30px] w-full">
+            {
+              aboutUs.activities.map((item) => {
+                return (
+                  <div className="flex flex-col text-white-2 gap-3">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)